test/app/components/product_component.rb

```diff
@@ -1,8 +1,10 @@
 # frozen_string_literal: true
 
 class ProductComponent < ViewComponent::Base
-  def initialize(product:, notice:)
+  def initialize(product:, notice:, count: nil, product_counter: nil)
     @product = product
-    @notice = notice
+    @notice  = notice
+    @count   = count
```

Comment:
What is `@count` used for in this example? It doesn't appear to be used in the template.

Reply:
Sorry, that's some stray code. Removing it